src/pal/src/safecrt/internal_securecrt.h

```diff
@@ -23,6 +23,13 @@
 #ifndef _INC_INTERNAL_SECURECRT
 #define _INC_INTERNAL_SECURECRT
 
+#ifndef __GLIBC__    // e.g. OS with musl-libc
+typedef unsigned long __uint64_t;
+typedef signed long __int64_t;
```

Comment:
It would be best to replace the few instances of `__int64_t` / `__uint64_t` with just `int64_t` / `uint64_t`so that we do not have to every worry about it again (the HP IA64 code where this is used can be just deleted).